apps/roam/src/utils/getDiscourseNodeType.ts

```diff
@@ -0,0 +1,48 @@
+import getPageTitleByPageUid from "roamjs-components/queries/getPageTitleByPageUid";
+import getDiscourseNodes, { type DiscourseNode } from "./getDiscourseNodes";
+import "core-js/proposals/regexp-escaping";
+import format from "date-fns/esm/fp/format/index.js";
+
+// eslint-disable-next-line
+const reEscape = (RegExp as any).escape as (str: string) => string;
+
+const SUB_RE = new RegExp(/\\\{\w+\\\}/g);
+
+export const formatToRegexpText = (format: string): string =>
+  reEscape(format.trim()).replaceAll(SUB_RE, ".*");
+
+export const getDiscourseNodeTypeByTitle = (
```

Comment:
As noted in another comment, let's use `findDiscourseNode` to be consistent with the code base.